go/libraries/doltcore/doltdb/doltdb.go

```diff
@@ -209,6 +213,11 @@ func (ddb *DoltDB) WriteEmptyRepoWithCommitMetaGeneratorAndDefaultBranch(
 		return err
 	}
 
+	rv, err = ddb.ciTablesCreator.CreateCIConfigTables(ctx, rv)
```

Comment:
You don't want to do this. The pattern you're using in `database.go`, which is correct, is that this table always implicitly exists, but never gets written to the root value unless you put data in it..